docs/build/qubit-order.mdx

```diff
@@ -5,35 +5,38 @@ description: How endianness works in Qiskit
 
 # Order of qubits in Qiskit
 
-While most physics textbooks represent an $n$-qubit system as the tensor product $Q_0\otimes Q_1 \otimes ... \otimes Q_{n-1}$, where $Q_j$ is the $j^{\mathrm{th}}$ qubit, Qiskit uses the inverse order, that is, $Q_{n-1}\otimes ... \otimes Q_1 \otimes Q_{0}$. As explained in [this video](https://www.youtube.com/watch?v=EiqHj3_Avps) from [Qiskit's YouTube channel](https://www.youtube.com/@qiskit), this is done to follow the convention in classical computing, in which the $n^{\mathrm{th}}$ bit or most significant bit (MSB) is placed on the left (with index 0) while the least significant bit (LSB) is placed on the right (index $n-1$). This ordering convention is called little-endian while the one from the physics textbooks is called big-endian.
+While most physics textbooks represent an $n$-qubit system as the tensor product $Q_0\otimes Q_1 \otimes ... \otimes Q_{n-1}$, where $Q_j$ is the $j^{\mathrm{th}}$ qubit, Qiskit uses the inverse order, that is, $Q_{n-1}\otimes ... \otimes Q_1 \otimes Q_{0}$. 
 
-This means that if we have, for example, a 3-qubit system with qubit 0 in state $|1\rangle$ and qubits 1 and 2 in state $|0\rangle$, Qiskit would represent this state as $|001\rangle$ while most physics textbooks would represent this state as $|100\rangle$.
+As explained in [this video](https://www.youtube.com/watch?v=EiqHj3_Avps) from [Qiskit's YouTube channel](https://www.youtube.com/@qiskit), this is done to follow the convention in classical computing, in which the $n^{\mathrm{th}}$ bit or most significant bit (MSB) is placed on the left (with index 0), while the least significant bit (LSB) is placed on the right (index $n-1$). This ordering convention is called *little-endian*, while the one from the physics textbooks is called *big-endian*.
```

Comment:
> As explained in \[this video\](https://www.youtube.com/watch?v=EiqHj3_Avps) 

Sadly that video is actually not quite right. I'll read the rest of the page tomorrow but I know for sure the video is wrong for reasons [mentioned in a previous PR on the topic](https://github.com/Qiskit/qiskit/pull/10205#discussion_r1221646250).

Reply:
Sorry – I had to sit down properly and get my head round it again. The video isn't as bad as I remember; I don't think I say anything technically incorrect. I'm ok with leaving it in; it doesn't really explain _why,_ but it does give useful examples of how the circuit corresponds to the measurement result.

Reply:
My main comment here is that I'd personally remove the discussion about the word "endian" and the terms "most significant" and "least significant" (and MSB and LSB).  The term "significance" only really makes sense when you're drawing the analogy to classical computing that Qiskit does with its ordering conventions, because _everybody_ writes integers with the "most significant" bit on the left in English.  Us using "endianness" is needlessly confusing, because the typical use of the word in classical computing actually refers to something completely unrelated.

Reply:
I've removed the discussion about endian and MSB/LSB -- lmk if more is needed! Thanks!